src/puppetlabs/puppetdb/helpers.clj

```diff
@@ -0,0 +1,89 @@
+(ns puppetlabs.puppetdb.helpers
```

Comment:
"helpers" seems a little generic to me for what this is, but maybe "cli" doesn't fit right either since we have cli tools that aren't in here. Maybe "cli-services"? "cli-helpers"? A more radical idea would be mounting this on the metadata service, although I could see that making less sense if this is ultimately going to house benchmark as well.